apps/storybook-react/stories/Snackbar.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { Snackbar, SnackbarProps, Button } from '@equinor/eds-core-react'
+import { Story, Meta } from '@storybook/react'
+
+const { SnackbarAction } = Snackbar
+
+export default {
+  title: 'Components/Snackbar',
+  component: Snackbar,
+  subcomponents: { SnackbarAction },
+} as Meta
+
+export const Default: Story<SnackbarProps> = (args) => {
+  const { open } = args
+  const [visible, setVisible] = useState(open)
```

Comment:
You can remove `const { open } = args` here and let the user trigger only at button click, that way the user can play with the other `Snackbar` examples too without interference... But it's weird how we get errors on it because this method has worked before 😕

Reply:
That is the my main issue - how do we know that we are not just masking errors by just removing code that doesn't work? And then I'll have to turn off the control for open as well 😬 

For the record, I'm doing the same thing with the Progress indicators - but because they is a single component with a single type it just works. But obviously not with the composed component approach

Reply:
`Open` is right there! 
![Screenshot 2020-11-03 at 08 34 45](https://user-images.githubusercontent.com/31915755/97959346-75563b80-1daf-11eb-90f0-6012b808bcf7.png)
🤔

Reply:
I also did it on Popover with subcomponents